src/index.ts

```diff
@@ -316,12 +344,5 @@ function constructRequest(har, opts = { userAgent: false, files: false, multipar
 
   options.headers = headers;
 
-  return new Request(`${url.split('?')[0]}${querystring ? `?${querystring}` : ''}`, options);
-}
-
-function fetchHar(har, opts = { userAgent: false, files: false, multipartEncoder: false }) {
-  return fetch(constructRequest(har, opts));
+  return fetch(`${url.split('?')[0]}${querystring ? `?${querystring}` : ''}`, req);
```

Comment:
Should this be `request` instead of `req`?

Reply:
should be `options` actually. fixed now